src/content/components/FlameChartCanvas.css

```diff
@@ -7,3 +7,7 @@
   top: 0;
   left: 0;
 }
+
+.flameChartCanvasTooltip {
+  padding: 5px;
+}
```

Comment:
just wondering why it's different for this tooltip ? I couldn't really debug and find out, because of the disappearing tooltip :)

Please add a comment explaining why, but I'd rather have something that works for all tooltips.